variables.tf

```diff
@@ -17,54 +17,60 @@
 # ---------------------------------------------------------------------------------------------------------------------
 
 variable "all_members_team_name" {
-  description = "The name of the team that contains all members of the organization."
+  description = "(Optional) The name of the team that contains all members of the organization."
   type        = string
   default     = null
 }
 
 variable "all_members_team_visibility" {
-  description = "The level of privacy for the team. Must be one of `secret` or `closed`."
+  description = "(Optional) The level of privacy for the team. Must be one of `secret` or `closed`."
```

Comment:
```suggestion
  description = "(Optional) The level of privacy for the team. Must be one of `secret` or `closed`. Default is \"secret\""
```

Reply:
We don't need that as the default value is already set. No reason to repeat what your IDE is showing you already anyways..

Reply:
Terradoc will create variables.tf not read it..

Reply:
Yes correct, still we don't need it in the description as well.